src/Simplex/Chat/Controller.hs

```diff
@@ -624,6 +626,29 @@ instance ToJSON ChatResponse where
   toJSON = J.genericToJSON . sumTypeJSON $ dropPrefix "CR"
   toEncoding = J.genericToEncoding . sumTypeJSON $ dropPrefix "CR"
 
+-- reduce to 4 result types, add connection type to result?
```

Comment:
I think link type should be top level, action bottom level, so the other way around?

Btw, we currently allow connecting own group link in incognito mode as a workaround to incognito groups. We can't kill this "feature" (obviously a more straightforward "incognito" group would be better).

Reply:
it should probably fall under "known" and allow to connect